dns/route53/dns_test.go

```diff
@@ -4,77 +4,55 @@ import (
 	"testing"
 
 	"net"
+	"fmt"
+	"os"
 
 	"github.com/aws/aws-sdk-go/aws"
-	route53Client "github.com/aws/aws-sdk-go/service/route53"
 	log "github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestEditResourceRecords(t *testing.T) {
```

Comment:
Please put the move of `TestEditResourceRecords` in a separate commit.

Reply:
Would you mind elaborating on your reasoning? If it is t make it visible or explain the reason behind this, would you not agree that the commit message already makes it clear?

Reply:
It would allow reviewing the additional tests added individually - currently it just drowns in the noise.